packages/contracts-bedrock/contracts/dispute/FaultDisputeGame.sol

```diff
@@ -0,0 +1,394 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import { IDisputeGame } from "./interfaces/IDisputeGame.sol";
+import { IVersioned } from "./interfaces/IVersioned.sol";
```

Comment:
This looks unused, contract impl should inherit

Reply:
`IVersioned` must be in scope for the `@inheritdoc` natspec tag: https://github.com/ethereum-optimism/optimism/pull/5984/files/a5770fe20e6ecf295d964a62741142c7e1e68df3#diff-3fdd2c2fb84eee383095af3282dfbba9a5563bc58957cdedda64ac85808e788fR389